app/models/admin.rb

```diff
@@ -1,3 +1,4 @@
+# class admin
```

Comment:
@abapolov can you please be more verbose in comments? `model for admin users` or something like this, at least.